libraries/inverse/hpiFit/sensorset.cpp

```diff
@@ -0,0 +1,151 @@
+#include <fiff/fiff_ch_info.h>
+
+//=============================================================================================================
+// QT INCLUDES
+//=============================================================================================================
+
+#include <QCoreApplication>
+
+//=============================================================================================================
+// EIGEN INCLUDES
+//=============================================================================================================
+
+#include <Eigen/Dense>
+
+//=============================================================================================================
+// USED NAMESPACES
+//=============================================================================================================
+
+using namespace INVERSELIB;
+using namespace FIFFLIB;
+using namespace FWDLIB;
+using namespace Eigen;
+
+//=============================================================================================================
+// DEFINE GLOBAL METHODS
+//=============================================================================================================
+
+//=============================================================================================================
+// DEFINE MEMBER METHODS
+//=============================================================================================================
+
+SensorSet::SensorSet(const FwdCoilSet::SPtr pFwdSensorSet)
+{
+    if(pFwdSensorSet!=nullptr) {
+        convertFromFwdCoilSet(pFwdSensorSet);
+    }
+}
+
+//=============================================================================================================
+
+void SensorSet::convertFromFwdCoilSet(const QSharedPointer<FWDLIB::FwdCoilSet> pFwdSensorSet)
+{
+    m_ncoils = pFwdSensorSet->ncoil;
+    m_np = pFwdSensorSet->coils[0]->np;
+    initMatrices(m_ncoils,m_np);
+
+    // get data froms Fwd Coilset
+    for(int i = 0; i < m_ncoils; i++){
+        FwdCoil* coil = (pFwdSensorSet->coils[i]);
+        MatrixXd matRmag = MatrixXd::Zero(m_np,3);
+        MatrixXd matCosmag = MatrixXd::Zero(m_np,3);
+        RowVectorXd vecW(m_np);
+
+        m_r0(i,0) = coil->r0[0];
+        m_r0(i,1) = coil->r0[1];
+        m_r0(i,2) = coil->r0[2];
+
+        m_ez(i,0) = coil->ez[0];
+        m_ez(i,1) = coil->ez[1];
+        m_ez(i,2) = coil->ez[2];
+
+        for (int p = 0; p < m_np; p++){
+            m_w(i*m_np+p) = coil->w[p];
+            for (int c = 0; c < 3; c++) {
+                matRmag(p,c)   = coil->rmag[p][c];
+                matCosmag(p,c) = coil->cosmag[p][c];
+            }
+        }
+
+        m_cosmag.block(i*m_np,0,m_np,3) = matCosmag;
+        m_rmag.block(i*m_np,0,m_np,3) = matRmag;
+    }
+    m_tra = MatrixXd::Identity(m_ncoils,m_ncoils);
+}
+
+//=============================================================================================================
+
+void SensorSet::initMatrices(int m_ncoils, int m_np)
```

Comment:
do not use m_ for function variables. Also consider making the int const

Reply:
this happens when you do "Replace All"